tests/testthat/test-model.R

```diff
@@ -183,6 +183,68 @@ test_that("sample() method works with init file", {
   expect_sample_output(mod$sample(data = data_file_r, init = init_file))
 })
 
+test_that("sample() method works with provided inv_metrics", {
+  skip_on_cran()
+
+  inv_metric_vector <- array(1, dim = c(1))
+  inv_metric_matrix <- matrix(1, nrow = 1, ncol = 1)
+
+  inv_metric_vector_json <- test_path("resources", "data", "bernoulli.inv_metric.diag_e.json")
+  inv_metric_matrix_json <- test_path("resources", "data", "bernoulli.inv_metric.dense_e.json")
+
+  inv_metric_vector_r <- test_path("resources", "data", "bernoulli.inv_metric.diag_e.data.R")
+  inv_metric_matrix_r <- test_path("resources", "data", "bernoulli.inv_metric.dense_e.data.R")
```

Comment:
I don't actually see these files in `tests/resources/data` so using them in tests below isn't going to work. Is the intention to save some inverse metrics in those files and then make sure they can be used successfully as the `metric_file` argument?